README.md

```diff
@@ -63,15 +69,15 @@ Most dependences can be installed using `apt-get install`:
 
 `sudo apt-get install g++ git automake libtool libgc-dev bison flex libgmp-dev libboost-dev pkg-config python python-scapy python-ipaddr tcpdump`
 
-An exception is Protocol Buffers; we need version 3.0, which is not available
```

Comment:
Let's mention the 3.0.2 version on the dependency list. We do put the version for Python.